components/input/drillTarget.js

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet, View } from "react-native";
+import { Text } from "react-native-paper";
+
+export default function DrillTarget({ drillTitle, distanceMeasure, value }) {
+  const getInstructionDisplay = (drill) => {
+    switch (drill) {
```

Comment:
Maybe this should be stored in firebase instead of here

Reply:
This should be discriminated based on the requirement instead of drill name, because multiple drill names can have the same requirements

Reply:
moved this to firebase and added a "prompt" field to the requirement